player/command.c

```diff
@@ -1695,7 +1689,7 @@ static int mp_property_ao_volume(void *ctx, struct m_property *prop,
         return M_PROPERTY_OK;
     }
     }
-    return M_PROPERTY_NOT_IMPLEMENTED;
+    return mp_property_generic_option(mpctx, prop, action, arg);
```

Comment:
Shouldn't this still be `M_PROPERTY_NOT_IMPLEMENTED`?

Reply:
done